Sources/FluentSQL/DatabaseQuery+SQL.swift

```diff
@@ -89,3 +89,19 @@ extension DatabaseQuery.Sort {
         .custom(expression)
     }
 }
+
+extension FieldKey: SQLExpression {
+    public func serialize(to serializer: inout SQLSerializer) {
+        switch self {
+        case .id:
+            serializer.write("id")
+        case .aggregate:
+            serializer.write("aggregate")
+        case .prefix(let prefix, let key):
+            prefix.serialize(to: &serializer)
```

Comment:
Is a prefix assumed to contain any "separator" needed between itself and the key? When does the `.prefix` case actually get used?

Reply:
Nope, it just means "concat these two things together"

Reply:
Love how GH doesn't notify me for comment edits so I see the email and don't realize you've already answered the other question 😆.